commit c23db208d9bb5472bb278ed00bbcff24f23cc18e
Merge: eafbc5e 1002eb6
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Sun Mar 6 15:52:57 2016 -0800

    Merge pull request #3101 from thinkyhead/rc_fix_stepper_func
    
    Update name of st_get_axis_position_mm in stepper.h

diff --git a/Marlin/stepper.h b/Marlin/stepper.h
index 4621194..8eb7016 100644
--- a/Marlin/stepper.h
+++ b/Marlin/stepper.h
@@ -66,8 +66,8 @@ void st_set_e_position(const long& e);
 // Get current position in steps
 long st_get_position(uint8_t axis);
 
-// Get current position in mm
-float st_get_position_mm(AxisEnum axis);
+// Get current axis position in mm
+float st_get_axis_position_mm(AxisEnum axis);
 
 // The stepper subsystem goes to sleep when it runs out of things to execute. Call this
 // to notify the subsystem that it is time to go to work.
